helpers/ifrauBackdropService.js

```diff
@@ -0,0 +1,9 @@
+export async function getIfrauBackdropService(ifrauBackdropService) {
```

Comment:
Since this method can find nothing, should we call it something like `tryGetIfrauBackdropService()`? Then it can return `null` is there's no `ifrauclient` (instead of `undefined` right now).